### ISSUE TITLE
Message not shown in a message thread after sending

### ISSUE THREAD

**Issue by pprucia** [wireapp/wire-android#603]
I was sending a message to someone on wire. After sending it doesn't show up in message thread immediately. I went back to recent contacts and then enter the thread back, still, message wasn't there. It appeared only after killing the app and launching it again.

[wire-logs.txt](https://github.com/wireapp/wire-android/files/773511/wire-logs.txt)

Wire version: 2.27.314

**Comment by someoneEsle** [comment]
duplicate of #595
